types/rdfjs__serializer-jsonld-ext/index.d.ts

```diff
@@ -19,10 +19,10 @@ declare namespace Serializer {
     }
 }
 
-declare class Serializer implements Sink {
+declare class Serializer<Q extends BaseQuad = Quad> implements Sink<Q> {
     constructor(options?: Serializer.SerializerOptions);
 
-    import(stream: Stream, options?: Serializer.SerializerOptions): EventEmitter;
+    import(stream: Stream<Q>, options?: Serializer.SerializerOptions): EventEmitter;
```

Comment:
All extensions of `@rdfjs/sink` return a `Stream`?

Reply:
I don't understand

Reply:
Sorry, mixed terminologies. Meant a `Readable` stream. But have just looked at the code and see `read` is a dummy function so no need to change it here.